src/Features/Core/Portable/ExternalAccess/UnitTesting/API/UnitTestingSearchHelpers.cs

```diff
@@ -138,31 +139,36 @@ private static void GetNameAndArity(string typeName, out string typeNameWithoutA
             var index = await TopLevelSyntaxTreeIndex.GetRequiredIndexAsync(document, cancellationToken).ConfigureAwait(false);
             foreach (var info in index.DeclaredSymbolInfos)
             {
-                // Fast check to see if this looks like a candidate.
-                if (info.TypeParameterCount == symbolArity &&
-                    info.Name == symbolName)
+                // Fast checks to see if this looks like a candidate.
+
+                // In non-strict mode, allow the type-parameter count to be mismatched.
+                if (query.Strict && info.TypeParameterCount != symbolArity)
+                    continue;
+
+                if (info.Name != symbolName)
```

Comment:
For VB the name could theoretically be different case. Is it possible to take that into account here?

Reply:
Actually never-mind, we'll probably require that the supplied name should match the casing present in the code in such cases. But one potential option may be to allow different casing for VB when strict is false.

Reply:
that seems like it would be *very* odd for the name to not match.  but sure, i can add that :)

Reply:
Yup doing that with strict=true would certainly be odd. But it would be nice to find closest match with strict=false :)

Note that I am only proposing with behavior when strict=off and also only when the language is VB :)